lib/transport/datagouvfr/authentication.ex

```diff
@@ -9,9 +9,10 @@ defmodule Transport.Datagouvfr.Authentication do
 
   # Public API
 
-  def client do
+  def client(token \\ nil) do
```

Comment:
Why is client defined here and not in ```Client``` ?

Reply:
It's a leftover of the old implementation. I'll move it.

Reply:
Actually it's part of the Authentication strategy, it can be moved easily.
OAuth2 expect `get_token!` and `get_token` to be part of the class generate the client